src/Cake.Common.Tests/project.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "version": "0.17.0-*",
+  "version": "0.16.2-*",
```

Comment:
You shouldn't change the version in the json files. Apart from these version changes, LGTM.